src/v1/imports/carts/grassdoor.ts

```diff
@@ -0,0 +1,88 @@
+import { QueryStringContext } from "../../../shared/types";
+
+const greendoorTracker = ({ appId, retailId }: Pick<QueryStringContext, "appId" | "retailId">) => {
+  const dataLayer = window.dataLayer || [];
+
+  function onDataLayerChange() {
+    const data = window.dataLayer.slice(-1)[0]; // Gets the newest array member of dataLayer
+
+    console.log(data);
```

Comment:
I must have forgot to remove this 😅

Reply:
Good catch, I honestly did not even see that